File: services/discount-service/src/main/java/com/fiji/discountservice/controller/DiscountResponse.java
package com.fiji.discountservice.controller;

import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class DiscountResponse {
    private Long id;

    private String name; // Название предложения

    private String company; // Название предложения

    private String description; // Процент скидки

    private LocalDateTime startDate; // Дата начала предложения

    private LocalDateTime endDate; // Дата окончания предложения

    private String discountCover;

    private boolean active; // Активно ли предложение
}

File: fiji-ui/src/app/services/discount/models/discount-response.ts
export interface DiscountResponse {
  id?: number;
  name?: string; // Название предложения
  company?: string; // Название компании
  description?: string; // Описание предложения
  startDate?: string; // Дата начала (в формате ISO 8601)
  endDate?: string; // Дата окончания (в формате ISO 8601)
  discountCover?: string; // Обложка скидки
  active?: boolean; // Активно ли предложение
}
